.github/workflows/ci.yml

```diff
@@ -252,5 +252,47 @@ jobs:
       functest: true
       kattest: false
       nistkattest: false
-      cbmc: true
+      cbmc: 2
```

Comment:
This is rather ugly. Can you split this into a boolean and an optional parameter named `cbmc_mlkem_k` or something like that?

Reply:
I thought we had a limit of 10 parameters for the workflow, but apparently it's already beyond that, so nevermind. I added a separate parameter.